docker/run-mysql.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-# The CloudSQL console simply states "MySQL 5.7" so we may not match the minor version number
-MYSQL_VERSION=5.7
+# The CloudSQL console simply states "MySQL 8.0" so we may not match the minor version number
```

Comment:
Is this comment still relevant now that we are specifying the minor version number?

Reply:
I think so, I am trying to specify the minor version number here, but there is no guaranty cloudSQL will use it 😅